examples/kanji_info.rs

```diff
@@ -0,0 +1,39 @@
+/*******************************************************************************
+* SPDX-License-Identifier: Apache-2.0 Refer to the file "LICENSE" for details.
+* example illustrating how to access kanji and reading information for a word
+*******************************************************************************/
+use jmdict::{KanjiElement, ReadingElement};
+
+fn main() {
+    let input = "一日";
+    let count = jmdict::entries()
+        .filter(|e| {
+            let mut result = false;
+            if e.kanji_elements().any(|k| k.text == input) {
+                result = true;
```

Comment:
You can avoid carrying the `result` variable through the entire function, and also avoid an extra level of indentation, by inverting the check and exiting early on failure:
```rust
            if e.kanji_elements().all(|k| k.text != input) {
                return false;
            }
            // ...
            true
```